src/guide/backend/index.md

```diff
@@ -8,3 +8,9 @@ You can use Passwordless.dev with any programming language by implementing calls
 - [PHP](php.md)
 - [Python 2](python2.md)
 - [Python 3](python3.md)
+
+Unofficial community SDKs:
+
+- [Rust (by davidzr)](https://github.com/davidzr/passwordless-rust)
```

Comment:
@Tyrrrz Should we add something related to our responsibility with regards to security and correct functioning of the SDKs?

Reply:
I think it's kind of implied in the word "unofficial". What do you suggest?